ntbs-service/DataAccess/ReferenceDataRepository.cs

```diff
@@ -183,17 +183,22 @@ public async Task<List<string>> GetPhecCodesMatchingRolesAsync(IEnumerable<strin
 
         public IQueryable<TBService> GetTbServicesQueryable()
         {
-            return _context.TbService;
+            return _context.TbService.OrderBy(s => s.Name);
```

Comment:
Should we put the ordering in one place (`GetTbServicesQueryable`), and refactor the other methods to go through this?

Also do we have suitable indexing to meet this requirement performantly?